Initiating the website (#1)
RConsortium/asa-biop-swe-wg: join_us.qmd

---
  title: "Join Us"
---

>   There are multiple ways to get involved with the ASA BIOP SWE WG

Reviewer: ```suggestion
  There are multiple ways to get involved with the ASA BIOP SWE WG:
```

>   * Join as a member, who attend the ASA BIOP SWE WG meetings

Reviewer: ```suggestion
  * Join as a member, who attends the ASA BIOP SWE WG meetings.
```

>   * Join as a task force member, who actively contributes to one or several task force(s)

Reviewer: ```suggestion
  * Join as a task force member, who actively contributes to one or several task force(s).
```

>   If you are interested to join the ASA BIOP SWE WG, please contact us **[to add contact info]**

Reviewer: ```suggestion
  If you are interested to join the ASA BIOP SWE WG, please contact us:
  [Daniel Sabanes Bove](mailto:daniel.sabanes_bove@roche.com) or [Ya Wang](mailto:ya.wang10@gilead.com)
```
